src/index.js

```diff
@@ -0,0 +1,215 @@
+
+const extendStringClass = {
+  /**
+   * Returns true if the string contains vowels.
+   * @return {Boolean}
+   */
+
+  hasVowels() {
+    return /[aeiou]/i.test(this);
+  },
+  /**
+   * Returns an Array of the upper and lower case alphabets.
+   * @return {Array}
+   */
+
+  characters() {
+    return 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    .split('');
+  },
+  /**
+   * Returns the String in question but with all characters in upper case.
+   * @return {String}
+   */
+
+  toUpper() {
+    return this.replace(/[a-z]/g, m =>
+      ''.characters()[''.characters().indexOf(m) + 26]
```

Comment:
**Code quality - Critical**
It is not optimal that you have to create a string instance to be able to assess the `characters` string. You are better off declaring it as a constant.